lib/internal/test_runner/test.js

```diff
@@ -176,22 +176,118 @@ function testMatchesPattern(test, patterns) {
 }
 
 class TestPlan {
-  constructor(count) {
-    validateUint32(count, 'count');
+  #waitIndefinitely = false;
```

Comment:
Just FYI - this class is internal to this file. So using private properties doesn't hurt, but also doesn't buy us anything.

Reply:
If there aren’t any performance penalties in using private properties, I think they can help prevent the exposure of internal properties across different functionalities (even within internals).
Btw, I don’t have strong opinions on this, so up to you 😁